### PR TITLE
Issue templates

### LINKED ISSUE #2: Update issue template
Update Flow.Launcher\ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by Kalaschnik**
- I added three templates (i.e., bug/issue report, feature suggestion, and code review)
- I deleted the old `ISSUE_TEMPLATE.md`
resolves #2

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F41E Bug report"
```
Comment (Kalaschnik) [inline .github/ISSUE_TEMPLATE/bug-report.md:2]: This entity corresponds to: 🐞

File: .github/ISSUE_TEMPLATE/code-review.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F50E Code Review"
```
Comment (Kalaschnik) [inline .github/ISSUE_TEMPLATE/code-review.md:2]: Will translate to: "🔍 Code Review"

**Comment by JohnTheGr8** [comment]
Do we want every issue to start with an emoji? The old man in me thinks that it will just create visual noise, and that the labels already solve the same problem...

**Comment by Kalaschnik** [comment]
> Do we want every issue to start with an emoji? The old man in me thinks that it will just create visual noise, and that the labels already solve the same problem...

I agree, labels even have more functions, however, one need to assign a label to every issue. Emoji would pre suggest a visual indication (unless removed by the issue author). Any other voices?

**Comment by JohnTheGr8** [comment]
> 
> I agree, labels even have more functions, however, one need to assign a label to every issue. Emoji would pre suggest a visual indication (unless removed by the issue author). Any other voices?

Can't the labels be assigned automatically from the template?

**Comment by Kalaschnik** [comment]
> > I agree, labels even have more functions, however, one need to assign a label to every issue. Emoji would pre suggest a visual indication (unless removed by the issue author). Any other voices?
> 
> Can't the labels be assigned automatically from the template?

This indeed works! I was note aware of that. I will remove the emojis and map the issue type like so:
Issue Type: Bug Report → bug (label)
Issue Type: Feature Request → enhancement (label)
Issue Type: Code Review → review (label, I just created it)

 @Flow-Launcher/team , does that work for you?

**Comment by jjw24** [comment]
Hey is code review label as in code/feature discussion?

Also do we want a type for the Question/Discussion issues?

**Comment by Kalaschnik** [comment]
> Hey is code review label as in code/feature discussion?

Not quite sure what you mean

> Also do we want a type for the Question/Discussion issues?

Good idea to add a Type for Question/Discussion. I would add it as well.

Updated:
Issue Type: `Bug Report` → `bug `(label)
Issue Type: `Feature Request` → `enhancement` (label)
Issue Type: `Code Review` → `code review` (label)
Issue Type: `Question/Discussion` → `question/discussion` (label)

**Comment by jjw24** [comment]
> > Hey is code review label as in code/feature discussion?
> 
> Not quite sure what you mean
> 
> > Also do we want a type for the Question/Discussion issues?
> 
> Good idea to add a Type for Question/Discussion. I would add it as well.
> 
> Updated:
> Issue Type: `Bug Report` → `bug `(label)
> Issue Type: `Feature Request` → `enhancement` (label)
> Issue Type: `Code Review` → `code review` (label)
> Issue Type: `Question/Discussion` → `question/discussion` (label)
> 
> Current Labels are here btw:
> https://github.com/Flow-Launcher/Flow.Launcher/issues/labels

What would we use the code review for?

**Comment by jjw24** [comment]
> > Hey is code review label as in code/feature discussion?
> 
> Not quite sure what you mean
> 

All good haha ignore me.

**Comment by Kalaschnik** [comment]
Should I also rename the label _enhacement_ to _feature request_, so that labels and issue templates share the same name?

**Comment by jjw24** [comment]
> Should I also rename the label _enhacement_ to _feature request_, so that labels and issue templates share the same name?

I think we keep it as enhancement, not always PR is feature request but feature request is also enhancement

**Comment by jjw24** [comment]
Is this pr ready rock?

**Comment by Kalaschnik** [comment]
> Is this pr ready to rock?

Yep, addressed the discussed changes.

File: .github/ISSUE_TEMPLATE/discussion.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: "⁉️ Question/Discussion"
```
Comment (jjw24) [inline .github/ISSUE_TEMPLATE/discussion.md:2]: should this be consistent with the other template- with out the '!?'

**Comment by jjw24** [comment]
> > Is this pr ready to rock?
> 
> Yep, addressed the discussed changes.

Legendary

**Comment by Kalaschnik** [comment]
I think GitHub needs IssueTemplates in master.

**Comment by JohnTheGr8** [comment]
@Kalaschnik yep, they'll be in master soon when we merge #101

**Pull request merged**